core/src/main/scala/monocle/Traversal.scala

```diff
@@ -38,7 +38,7 @@ abstract class PTraversal[S, T, A, B] extends Serializable { self =>
 
   /** map each target to a Monoid and combine the results */
   @inline final def foldMap[M: Monoid](f: A => M)(s: S): M =
-    modifyF[Const[M, ?]](a => Const(f(a)))(s).getConst
+    modifyF[({type λ[α] = Const[M, α]})#λ](a => Const(f(a)))(s).getConst
```

Comment:
why not, you're right is the most interesting usage of kind projector. Adding a comment could help reader